### ISSUE TITLE
Write text in image?

### ISSUE THREAD

**Issue by natrius** [borisdayma/dalle-mini#312]
Is it possible to write something? I tried `Banana with the word frog written on it` and the output is anything but text
![image](https://user-images.githubusercontent.com/7243447/202698445-64a2dc96-b33d-4128-8a1b-03addd756983.png)

I tried `Man wearing a t-shirt that says I like cake` as well as `Man wearing a t-shirt that says "I like cake"`
![image](https://user-images.githubusercontent.com/7243447/202698666-6c95c581-a158-46f1-90af-248354edafc2.png)

But it seems dalle-mini is not able to work out on how to create text. Or is there a way to get the desired output?

**Comment by borisdayma** [comment]
Yeah the model is not very good at writing text, probably need a bigger model

**Comment by PrathyushaModala** [comment]
@natrius  Can I work on this issue??

**Comment by natrius** [comment]
Why are you asking me? Go for it! :D
